libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/OAuth2.java

```diff
@@ -768,13 +772,18 @@ public static class TokenEndpointResponse {
         public String contentDomain;
         public String contentSid;
         public String csrfToken;
+        public String cookieClientSrc;
+        public String cookieSidClient;
+        public String sidCookieName;
 
         /**
          * Parameterized constructor built during login flow.
          *
          * @param callbackUrlParams Callback URL parameters.
+         * @param additionalOauthKeys Additional oauth keys.
          */
-        public TokenEndpointResponse(Map<String, String> callbackUrlParams) {
+        @VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
+        public TokenEndpointResponse(Map<String, String> callbackUrlParams, List<String> additionalOauthKeys) {
```

Comment:
Previously we had `public TokenEndpointResponse(Map<String, String> callbackUrlParams)` which get the `additionalOauthKeys` from `SalesforceSDKManager`.
Now we still have `public TokenEndpointResponse(Map<String, String> callbackUrlParams)` but there is also this one: `public TokenEndpointResponse(Map<String, String> callbackUrlParams, List<String> additionalOauthKeys)` used by the tests (so that the test does not have to set/unset `additionalOauthKeys` on SalesforceSDKManager.